### ISSUE TITLE
[BUG] Application opens twice when using right click > uninstall 

### ISSUE THREAD

**Issue by saied-elbach** [alienator88/Pearcleaner#68]
**Describe the bug:** <br>
When I right click on an application to uninstall it, PearCLeaner opens two windows for removing the chosen application twice.

**Steps to reproduce:**
1. Go to 'application'
2. Right Click on any program
3. Scroll down to PearCleaner Uninstall
4. See 2 windows of PearCleaner

**Expected behavior** <br>
PearCleaner should open only 1 window

**Info:**
 - OS: [15.0]
 - Pearcleaner Version: [3.7.3]

**Screenshots**

![Screenshot 2024-06-17 at 09 59 05](https://github.com/alienator88/Pearcleaner/assets/169826562/6e2bd716-44b4-467d-88c4-a92806fb07d8)

**Comment by alienator88** [comment]
I tried to cause this bug and I can't reproduce the issue on my end on multiple computers for the supported OS's my app works with.

I did notice your OS is Sequoia, this is a developer beta. I can't build for this version to support it until it's public release because it's too difficult to support as they usually have too many inconsistencies and bugs that eventually get squashed by the time its made public.

I will take a look at this issue once Sequoia is out for public use.
